pages_unit_tests/opening-hours.test.js

```diff
@@ -0,0 +1,364 @@
+        opening_day_wednesday: "Wednesday"
+      };
+      const wrapper = mount(
+        <OpeningHours
+          validatorErrors={{
+            opening_hours_wednesday: "invalid opening hours on wednesday"
+          }}
+          cumulativeFullAnswers={cumulativeFullAnswers}
+        />
+      );
+      const OpeningHoursField = wrapper.find(
+        "InputField#opening_hours_wednesday"
+      );
+      expect(OpeningHoursField.props().meta.error).toBe(
+        "invalid opening hours on wednesday"
+      );
+    });
+    it("renders correct error message for Thursday field", () => {
+      const cumulativeFullAnswers = {
+        opening_day_thursday: "Thursday"
+      };
+      const wrapper = mount(
+        <OpeningHours
+          validatorErrors={{
+            opening_hours_thursday: "invalid opening hours on thursday"
+          }}
+          cumulativeFullAnswers={cumulativeFullAnswers}
+        />
+      );
+      const OpeningHoursField = wrapper.find(
+        "InputField#opening_hours_thursday"
+      );
+      expect(OpeningHoursField.props().meta.error).toBe(
+        "invalid opening hours on thursday"
+      );
+    });
+    it("renders correct error message for Friday field", () => {
+      const cumulativeFullAnswers = {
+        opening_day_friday: "Friday"
+      };
+      const wrapper = mount(
+        <OpeningHours
+          validatorErrors={{
+            opening_hours_friday: "invalid opening hours on friday"
+          }}
+          cumulativeFullAnswers={cumulativeFullAnswers}
+        />
+      );
+      const OpeningHoursField = wrapper.find("InputField#opening_hours_friday");
+      expect(OpeningHoursField.props().meta.error).toBe(
+        "invalid opening hours on friday"
+      );
+    });
+    it("renders correct error message for Saturday field", () => {
+      const cumulativeFullAnswers = {
+        opening_day_saturday: "Saturday"
+      };
+      const wrapper = mount(
+        <OpeningHours
+          validatorErrors={{
+            opening_hours_saturday: "invalid opening hours on saturday"
+          }}
+          cumulativeFullAnswers={cumulativeFullAnswers}
+        />
+      );
+      const OpeningHoursField = wrapper.find(
+        "InputField#opening_hours_saturday"
+      );
+      expect(OpeningHoursField.props().meta.error).toBe(
+        "invalid opening hours on saturday"
+      );
+    });
+    it("renders correct error message for Sunday field", () => {
+      const cumulativeFullAnswers = {
+        opening_day_sunday: "Sunday"
+      };
+      const wrapper = mount(
+        <OpeningHours
+          validatorErrors={{
+            opening_hours_sunday: "invalid opening hours on sunday"
+          }}
+          cumulativeFullAnswers={cumulativeFullAnswers}
+        />
+      );
+      const OpeningHoursField = wrapper.find("InputField#opening_hours_sunday");
+      expect(OpeningHoursField.props().meta.error).toBe(
+        "invalid opening hours on sunday"
+      );
```

Comment:
As above. I would add one case which tests more than one day at once with different validation warnings. This gives confidence that no copy/paste mistakes have been made. From a white-box perspective, we know this not to be the case here. From a black-box perspective, it's good to cover in case of refactoring.